modules/cid_checker/variables.tf

```diff
@@ -0,0 +1,38 @@
+
+variable "bucket_name" {
+  description = "Name of S3 bucket."
+  type = string
+}
+
+variable "force_destroy" {
+  description = "If `true`, will destroy S3 bucket. Defaults to `false`."
+  type = bool
+  default = false
+}
+
+variable "tags" {
+  description = "Map of general common tags."
+  type = map(any)
+  default = {}
+}
+
+variable "oidc" {
+  description = "OpenID Connect endpoint URLs for cluster."
+  type = string
+}
+
+variable "name_prefix" {
+  description = "General name prefix for IAM."
+  type = string
+}
+
+variable "kubernetes_service_account" {
+  description = "Variables for kubernetes service account."
+  type = object({
+    namespace   = string
+  })
+    default = {
+      namespace        = "default"
+    }
+  }
+
```

Comment:
I'm not sure, what this variable is built for, and why we have this structure type.
If the main idea is around the namespace, then it should be a string. And if my guess is right, when we don't need to provide a default value for it. instead of this, I would suggest configuring requirements to get the namespace string from the init module directly.

Reply:
Was changed this object to single variable 
 with type "string".